Api-docs.md

```diff
@@ -0,0 +1,1949 @@
+        </tr>
+      </table>
+  
+    - URL : `/api/share/`
+  
+    - Fields :
+    
+      <table>
+        <tr>
+          <th>Field</th>
+          <th>Required</th>
+          <th>Readonly</th>
+          <th>Unique</th>
+          <th>Type</th>
+          <th>Default</th>
+          <th>Description</th>
+        </tr>
+        <tr>
+          <td>object_type</td>
+          <td>True</td>
+          <td>True</td>
+          <td>-</td>
+          <td>String</td>
+          <td>-</td>
+          <td>target type of the share object</td>
+        </tr>
+        <tr>
+          <td>object_id</td>
+          <td>True</td>
+          <td>True</td>
+          <td>-</td>
+          <td>int</td>
+          <td>-</td>
+          <td>Id of the target object</td>
+        </tr>
+        <tr>
+          <td>sharer_type</td>
+          <td>True</td>
+          <td>True</td>
+          <td>-</td>
+          <td>String</td>
+          <td>-</td>
+          <td>Sharer type of the share object</td>
+        </tr>
+        <tr>
+          <td>sharer_id</td>
+          <td>True</td>
+          <td>True</td>
+          <td>-</td>
+          <td>int</td>
+          <td>-</td>
+          <td>Id of the sharer object</td>
+        </tr>
+        <tr>
+          <td>relationship</td>
+          <td>-</td>
+          <td>-</td>
+          <td>-</td>
+          <td>Choice</td>
+          <td>-</td>
+          <td>Relationship of share. creator, admin, editor, viewer</td>
+        </tr>
+        <tr>
+          <td>shared_by</td>
+          <td>True</td>
+          <td>True</td>
+          <td>-</td>
+          <td>int</td>
+          <td>-</td>
+          <td>Id of the user that created the share object</td>
+        </tr>
+        <tr>
+          <td>user</td>
+          <td>-</td>
+          <td>True</td>
+          <td>-</td>
+          <td>Json</td>
+          <td>-</td>
+          <td>Display linked user only if sharer_type is user</td>
+        </tr>
+      </table>
+
+    - Fields Description:
+
+    	- Relationship Types:
+	 		- Creator : Have all permissions to edit survey and can also create another creater type share
+	 		- Admin : Have all permissions to edit survey but can only edit editor and viewer type shares
+	 		- Editor : Have some permissions to edit survey and can only edit viewer type shares
+	 		- Viewer : Can only view survey and survey details
+
+	 	- Sharer Types:
+	 		- user : share tareget evaluation/survey/customreport with user
+	 		- organisation : share tareget evaluation/survey/customreport with orgaisation
+
+	 	- Target Types:
+	 		- evaluation : user/organisation will get access to all survey's of shared evaluation
+	 		- suvrey : user/organisation will get access to a specific survey data
+	 		- customreport : user/organisation will get access to a specific custom minisite report data
+
+	 	- Tips:
+	 		- Targret and sharer are required when creating a share.
```

Comment:
Targret = "Target"

Might be good to run this doc through spell check.

Reply:
👍 
Thanks @chr15m